carsFilter/src/main/java/tkhal.filter.controller/CarsFilter.java

```diff
@@ -4,17 +4,25 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * @author Taras Khalak
```

Comment:
Will this information help other developer to understand what does this class do?

Reply:
done